hail/python/hailctl/dataproc/resources/init_notebook.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
oops, I bundled these changes accidentally. The init script is still not functional (did the one you PRed actually work to generate clusters?) but is closer.

Reply:
I didn't test it at all. All I did was startup dataproc without the original init script and see where everything was located.

Reply:
OK, sure. I'll get this stuff working next.